apps/ledger-live-mobile/src/screens/ReceiveFunds/01-SelectCrypto.tsx

```diff
@@ -50,7 +51,8 @@ export default function AddAccountsSelectCrypto({ navigation, route }: Props) {
   const { t } = useTranslation();
   const accounts = useSelector(flattenAccountsSelector);
 
-  const { currenciesByProvider, sortedCryptoCurrencies } = useGroupedCurrenciesByProvider();
+  const { currenciesByProvider, sortedCryptoCurrencies } =
+    useGroupedCurrenciesByProvider() as GroupedCurrencies;
```

Comment:
why do you need to force casting?

Reply:
**With the current implementation** simply because this "hook" has 2 return type with different object shape, so typescript won't be able to infer which one you are using. 
I'm not willing to add more refactoring to this `useGroupedCurrenciesByProvider` with generics as we will clean it up with `GroupedCurrencies`